scss/index.scss

```diff
@@ -0,0 +1,126 @@
+@import "imports/variables";
+
+@font-face {
+  font-family: Ketchum;
+  src: url('../fonts/Ketchum.otf');
+}
+
+body {
```

Comment:
box-sizing needs to go on all elements e.g 

```*``` {
box-sizing: border-box
}

Reply:
done in  e919a8b